app/services/analytics/builders.py

```
@@ -0,0 +1,447 @@
+        display_change_indicator=display_change,
+    )
+
+
+def build_analytics_dashboard_view(
+    snapshot_orm: Optional[object],
+    alerts_list: list,
+    events_list: list,
+    window_type: str,
+    window_start: datetime,
+    window_end: datetime,
+    class_id: str,
```

Ruff (0.16.1)

[warning] Unused function argument: `class_id`

(ARG001)